spec/game_runner_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+require_relative '../game_runner.rb'
+
+RSpec.describe GameRunner do
+  def grid_template
```

Comment:
There isn't any reason for this to be a separate method. Having it in the test itself will make it easier to understand!